website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,392 @@
+---
+title: Notifications
+---
+
+In the previous section, we learned how to develop forms. In this section we look at how to communicate feedback to users using notifications, particularly in response to user actions.
+
+To facilitate the use of notifications, Custom Applications come with a built-in notification system.
+
+# Notification system
+
+Notifications are an important way of providing feedback in response to user actions.
+For example submitting a form might succeed or fail. As a result the user should be notified about that in a way that is understandable and easy to find.
+
+Notifications are grouped by what we call "domains." A domain indicates where the notification should be displayed on the page (for instance `page` and `side`).
+
+Notifications are also of different types: `success`, `error`, `warning`, `info`. Depending on the specific use case you would choose the appropriate type. For example, if a form has been successfully submitted, you would show a "side" notification of kind `success`.
+
+Triggering notifications is done by using some custom React hooks. Please refer to the package [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) for implementation details.
+
+<Info>
+
+For defining the domain and the type of a notification, we recommend using the values exposed from `@commercetools-frontend/constants`.
```

Comment:
Since `domain` and `type` are properties to be used with the notifications code, I think it worths it to highlight them:
```suggestion
For defining the `domain` and the `type` of a notification, we recommend using the values exposed from `@commercetools-frontend/constants`.
```